src/Group.java

```diff
@@ -0,0 +1,64 @@
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Group {
+    private Student groupHead ;
+    private List<Student> groupMembers = new ArrayList<>();
+    private List<String> tasksForGroup = new ArrayList<>();
+
```

Comment:
Add structure `Map<String, List<Student>> compleatedTasks = new HashMap<>();` to mark compleaded tasks for students